### ISSUE TITLE
Refers to controller action that hasn't been mentioned yet. 

### ISSUE THREAD

**Issue by authorbeard** [learn-co-curriculum/sinatra-mvc-file-structure#7]
Under subhead: "views directory":

"We've filled in the views for you to display the dog Ralph we just created in our controller action. Feel free to take a look." 

Nothing of the sort has been filled out that I could find, and this is the first mention of Ralph. I Googled around and eventually figured out how to pass the last test on my own, but wasted a lot of time checking other lessons and trying to find these examples. Hadn't actually written any erb or incorporated MVCs at all to this point. Seems like the lesson's out of order or the order was changed at some point without the readme being updated to reflect that.

**Comment by victhevenot** [comment]
hey @authorbeard thanks so much for bringing this to our attention! Making a pull request now to update :)

**Comment by authorbeard** [comment]
No problem. Turns out it's like, two lessons later when all that's explained, but it's basic enough erb stuff that I was able to Google it. Had the same issue with the ActiveRecord section; that had a lab that came before two readmes that explained some basic startup stuff. Which isn't necessarily a bad way to do it, but some more clues somewhere at the beginning (like an additional test that alerts us we need to go find this other thing) would help a bit.
